tdrs-backend/tdpservice/settings/cloudgov.py

```diff
@@ -120,6 +122,26 @@ class CloudGov(Common):
         "Cache-Control": "max-age=86400, s-maxage=86400, must-revalidate",
     }
 
+    AWS_ELASTIC_ACCESS_KEY = os.getenv('AWS_ELASTIC_ACCESS_KEY', '')
+    AWS_ELASTIC_SECRET = os.getenv('AWS_ELASTIC_SECRET', '')
```

Comment:
these secrets are provided in the logs after binding the elastic service to the backend application in Cloud.Gov. They need to be added via environment variables to each of the deployed applications.

Reply:
Is there a chicken-egg problem here or should it be accessible via environment variables within the deployed env? You say they're provided in the logs -- cloud.gov's logstreamer or elasticsearch logs?